spec/acceptance/default_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+require 'spec_helper_acceptance_local'
+
+describe 'opensearch' do
+  context 'default parameter' do
+    it 'works' do
+      manifest = 'include opensearch'
+
+      apply_manifest(manifest, catch_failures: true)
+      apply_manifest(manifest, catch_changes: true)
+    end
```

Comment:
We have a helper for this in voxpupuli-acceptance:
```suggestion
    it_behaves_like 'an idempotent resource' do
      let(:manifest) do
        'include opensearch'
      end
    end
```

Reply:
fixed